### ISSUE TITLE
Error: Unknown dataset

### ISSUE THREAD

**Issue by ryhhtn** [salesforce/LayoutDETR#2]
Thanks for providing the interesting code.

I tried to [Data Preprocessing](https://github.com/salesforce/LayoutDETR#data-preprocessing), but got an error.
```
Error: Unknown dataset
```

As far as [this code](https://github.com/salesforce/LayoutDETR/blob/7da846caab00dd66a2b9c5804e8dcf57b163fe64/dataset_tool.py#L247-L253) is concerned, I always get an error even if I run it according to the documentation.
Is there a workaround?

**Comment by ningyu1991** [comment]
Yeah, the key string "ads_banner_collection_manual" needs to be within your data source path. You can modify the code at Line 248 in dataset_tool.py to skip this error.
